django/db/backends/sqlite3/features.py

```diff
@@ -64,6 +64,13 @@ class DatabaseFeatures(BaseDatabaseFeatures):
     insert_test_table_with_defaults = 'INSERT INTO {} ("null") VALUES (1)'
     supports_default_keyword_in_insert = False
 
+    # Indicates if the database backend supports conditional updates in bulk
+    # operations. True for backends that allow syntax like
+    # "INSERT INTO ... ON CONFLICT (...) DO UPDATE SET ... WHERE ...".
+    # Set to True for backends that support this feature.
+
+    supports_conditional_update = True
+
```

Comment:
We don't need to copy the comment here, it's in the base backend.

As before, let's call this `supports_update_conflicts_where` and move it up adjacent to `supports_update_conflicts`.